ld/src/ld/parsers/libunwind/DwarfInstructions.hpp

```diff
@@ -186,6 +187,11 @@ const char* DwarfInstructions<A,R>::parseCFIs(A& addressSpace, pint_t ehSectionS
                                       bool keepDwarfWhichHasCU,  bool forceDwarfConversion, bool neverConvertToCU,
                                       CFI_Atom_Info<A>* infos, uint32_t& infosCount, void* ref, WarnFunc warn)
 {
+    std::set<int> cuStartsSet;
```

Comment:
this could even be initilised with the right size using `cuCount` right?

Reply:
Set is based on red-black tree, so it don't need to pre-allocate space;
I changed it to unordered_set.